### DISCUSSION TITLE
copio la supuesta resolucion exacta y me sigue saltando incorrecto.

### DISCUSSION THREAD

**Discussion by Ivandionisio** [atralice/Curso.Prep.Henry#113] [Q&A]
Estoy en la seccion 03 de homework y poniendo exactamente la solucion- ya que despues de multiples intento no logre hallar la respuesta correcta por cuenta propia- me sigue saltando que es incorrecta, alguno sabe porque? muchas gracias!
subi una foto, es el ejercico de mayor de 18 y tambien me paso con el anterior

![bandicam 2021-08-27 17-54-19-435](https://user-images.githubusercontent.com/47585154/131187737-3a039620-f53b-4ece-ad0c-65437e16ba3a.jpg)

**Comment by VicoVir** [comment]
Hola, hay que respetar las mayúsculas osea no es allowed es Allowed, lo mismo con not allowed es Not allowed.
